lcov.info

```diff
@@ -0,0 +1,30 @@
+TN:
```

Comment:
Kindly get rid of this file and add it to .gitignore

Reply:
done thanks